addons/account/__init__.py

```diff
@@ -10,9 +10,16 @@
 
 SYSCOHADA_LIST = ['BJ', 'BF', 'CM', 'CF', 'KM', 'CG', 'CI', 'GA', 'GN', 'GW', 'GQ', 'ML', 'NE', 'CD', 'SN', 'TD', 'TG']
 
-def _auto_install_l10n(cr, registry):
+def _set_fiscal_country(env):
+    """ Sets the fiscal country on existing companies when installing the module,
+    defaulting it to country_id.
+    """
+    for company in env['res.company'].search([]):
+        company.account_tax_fiscal_country_id = company.country_id
```

Comment:
I don't think you need that anymore. Why not:
```
account_tax_fiscal_country_id = fields.Many2one(
    comodel_name='res.country',
    string="Fiscal Country",
    store=True, readonly=False,
    compute='_compute_account_tax_fiscal_country_id'
)

@api.depends('country_id')
def _compute_account_tax_fiscal_country_id(self):
   for company in self:
       company.account_tax_fiscal_country_id = company.country_id
```

Reply:
Because it won't initialize itself otherwise (the compute won't be called for existing companies). Though it's true that I could just initialize the compute in post_init_hook instead. I'll do that.